NOTICE

```diff
@@ -0,0 +1,20 @@
+AxoSyslog documentation
+Copyright 2023-2025 Axoflow Inc
+
+NOTICE file corresponding to the section 4d of The Apache License. You can use, copy, modify, and/or distribute the AxoSyslog documentation in compliance with section 4d of The Apache License.
```

Comment:
With the terms described by the apache license.

Reply:
Add a reference to the license file (full link pointing to the file in our GitHub)